Woodall_Robert/Assignments/multiples.py

```diff
@@ -0,0 +1,22 @@
+'''
+
+Part 1:
+Create a program that prints all the odd numbers from 1 to 1000. Use the for loop and don't use array to do this exercise.
+
+Part 2:
+Create another program that prints all the multiples of 5 from 5 to 1000000
+
+'''
+
+print "Printing odd values from 1 - 1000..."
+
+for counter in range(1, 1000):
+	if (counter % 2) != 0:
```

Comment:
Cool. Check out the official docs for the range() function. There might be an even easier way to do this..